From fdf7a68b6c618ebb0abad8fb6d41133a412dcd9d Mon Sep 17 00:00:00 2001
From: Zeng Zhaoming <b32542@freescale.com>
Date: Thu, 30 Jun 2011 17:50:59 +0800
Subject: ENGR00152364 MX6,GPIO: GPIO virtual irq tiggered with wrong irq
 number

GPIO virtual irq triggered with wrong irq number.

It is caused by we enable TZIC and set GPIO virtual irq start at 128.

Signed-off-by: Zeng Zhaoming <b32542@freescale.com>
---
 arch/arm/configs/imx6_defconfig |    1 -
 arch/arm/mach-mx6/Kconfig       |    1 -
 2 files changed, 2 deletions(-)

diff --git a/arch/arm/configs/imx6_defconfig b/arch/arm/configs/imx6_defconfig
index 93cae1a..10cc366 100644
--- a/arch/arm/configs/imx6_defconfig
+++ b/arch/arm/configs/imx6_defconfig
@@ -276,7 +276,6 @@ CONFIG_MACH_MX6Q_SABREAUTO=y
 #
 CONFIG_ISP1504_MXC=y
 # CONFIG_MXC_IRQ_PRIOR is not set
-CONFIG_MXC_TZIC=y
 CONFIG_MXC_PWM=y
 # CONFIG_MXC_DEBUG_BOARD is not set
 CONFIG_ARCH_MXC_IOMUX_V3=y
diff --git a/arch/arm/mach-mx6/Kconfig b/arch/arm/mach-mx6/Kconfig
index 389a9c0..0d9b07b 100644
--- a/arch/arm/mach-mx6/Kconfig
+++ b/arch/arm/mach-mx6/Kconfig
@@ -3,7 +3,6 @@ if ARCH_MX6
 config ARCH_MX6Q
 	bool
 	select USB_ARCH_HAS_EHCI
-	select MXC_TZIC
 	select ARCH_MXC_IOMUX_V3
 	select ARM_GIC
 	select IMX_HAVE_PLATFORM_IMX_UART
-- 
1.7.9.5

